Sources/CoreAardvark/PrivateCategories/NSFileHandle+ARKAdditions.m

```diff
@@ -32,8 +46,30 @@
 
 @implementation NSFileHandle (ARKAdditions)
 
++ (void)ARK_setPreventWritesAfterException:(BOOL)preventWritesAfterException
+{
+    // convert from ObjC BOOL (a `char` on some platforms) to stdbool
+    // see: Special Considerations on https://developer.apple.com/documentation/objectivec/bool
+    bool stdboolWritesAfterException = preventWritesAfterException;
```

Comment:
maybe this should be `bool stdboolWritesAfterException = !!preventWritesAfterException;`?

Reply:
Seems like a good idea (and it's not like this will get called with any kind of frequency).

Reply:
Oh, and this local var should be named `stdBool **Prevent** WritesAfterException` yes?

Reply:
oops yes, thanks for catching the missing `Prevent`. Will update to add the `!!` + fix this.

Reply:
updated, thanks again